CMSIS/Core/Include/core_ca.h

```diff
@@ -22,6 +16,10 @@
  * limitations under the License.
  */
 
+/*
+ * CMSIS Cortex-A Core Peripheral Access Layer Header File
+ */
+
```

Comment:
I'd remove the __CA_CMSIS_VERSION macros and instead include the cmsis_version.h file here as well.

Reply:
removed defines for CA version and added` #include "cmsis_version.h"`